src/supremm/xdmodaccount.py

```diff
@@ -1,4 +1,5 @@
 """ Implementation for account reader that gets data from the XDMoD datawarehouse """
+import sys #debug
```

Comment:
Please double check that you want to include this new line - it looks like it is left over from some debugging while developing the code.

Reply:
Resolved in https://github.com/ubccr/supremm/pull/259/commits/a50f9f32a7e63d5804f3f01844149b6cc2936be9